packages/website/pages/account/payment.js

```diff
@@ -193,7 +193,7 @@ export function getStaticProps() {
     props: {
       title: 'Payment',
       isRestricted: true,
-      redirectTo: '/login/',
+      redirectTo: '/login/?redirect_uri=/account/payment',
```

Comment:
I think this should just be `/account/payment`

Reply:
redirectTo is where the http response will redirect to when the end-user isn't authenticated. This page is served at /account/payment. If redirect_to=/account/payment, it would be an infinite redirect.

Reply:
Ah ok, I misunderstood what was happening there.